lambdas/data-migration2/src/executions.ts

```diff
@@ -97,13 +102,15 @@ export const migrateExecutions = async (
         migrationSummary.skipped += 1;
       } else {
         migrationSummary.failed += 1;
-        logger.error(
-          `Could not create execution record in RDS for Dynamo execution arn ${record.arn}:`,
-          error
-        );
+        errorMessage = `Could not create execution record in RDS for Dynamo execution arn ${record.arn}:`;
+        errorFile.push(`Error: ${error} ${errorMessage}`);
```

Comment:
Could this approach exceed the lambda/container's memory capacity for a reasonably large numbers of skips/errors (?) and if so I'm wondering if we should consider using a write stream or another defensive approach?

Reply:
+1. I think using a write stream would be safer.